Jenkinsfile

```diff
@@ -299,6 +300,19 @@ def job = {
                                 cmd += " -DnexusUrl=s3://staging-ksqldb-maven/maven"
                                 sh cmd
                             }
+                            sh """
+                                set +x
+                                TMP_GPG_PASS=\$(mktemp -t XXXgpgpass)
+                                echo "\${GPG_PASSPHRASE}" > "\${TMP_GPG_PASS}"
+                                bash confluent-process-packages.sh \
+                                    --bucket staging-ksqldb-packages \
+                                    --region us-west-2 \
+                                    --prefix '\$PACKAGE_TYPE/${config.ksql_db_packages_prefix}' \
+                                    --input-dir "\${PWD}/output" \
+                                    --sign-key-file "${gpg_packaging_key}" \
+                                    --passphrase-file "\${TMP_GPG_PASS}"
+                                rm -f "\${TMP_GPG_PASS}"
```

Comment:
would it be bad to leave this around? (e.g. if confluent-process-packages fails?)

Reply:
I'll add something for added protection. But it should be noted that Jenkins "throws" away spawned containers when the Job ends.